### PR TITLE
added in sugan branch

### PR DESCRIPTION
**Pull request by suganhillary**
Kindly check and merge mybranch

### CODE REVIEW

File: gitbranchchecking/gitbranchpractise.html
Comment (Eddi1995) [inline gitbranchchecking/gitbranchpractise.html:1]: ok seems good wait for sometime ill check with manager and let you know

**Comment by Eddi1995** [comment]
wait for sometimes ill check and let you know

**Comment by suganhillary** [comment]
Ok sir, please confirm me once it is merge, thanks

**Comment by Eddi1995** [comment]
Ok sure, thanks

**Comment by Eddi1995** [comment]
we check the files everything looks good, we will merge sugansupport to main branch, thanks

**Comment by suganhillary** [comment]
thanks machi

**Review by Eddi1995 — APPROVED** [review]
Ready to merge, thanks

**Pull request merged**